demos/supabase-simple-chat/pubspec.yaml

```diff
@@ -37,7 +37,7 @@ dependencies:
 
   supabase_flutter: ^1.10.25
   timeago: ^3.6.0
-  powersync: ^1.4.2
+  powersync: ^1.5.0
```

Comment:
Could you make this change in all the demos. Update them all to use `powersync: ^1.5.0` in their respective `pubspec.yaml` files. This is just in case someone tries running the demos without using `melos`

Reply:
Looks like the rest are all already on `^1.5.0`.